doc/functions.md

```diff
@@ -121,9 +129,22 @@ def times2 x := x * 2
 #eval 100 |> add1 |> times2
 #eval times2 <| add1 <| 100
 ```
-The result of the previous `#eval` commands is 202.
-The forward pipeline `|>` operator takes a function and an argument and return a value.
-In contrast, the backward pipeline `<|` operator takes an argument and a function and returns a value.
+The result of all three previous `#eval` commands is 202.
+
+So `x |> f` means the same as the same as `f x`, and it chains such that `x |> f |> g` is
```

Comment:
suggestion:

So `x |> f` means the same as the same as `f x`, and ~it chains such that~ `x |> f |> g` is

Reply:
I got this directly from `Notation.lean` - is there a problem with saying "it chains" ?